reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -305,7 +305,7 @@ class StaticHtmlReporter : Reporter {
                 }
             }
             td {
-                p { +ruleViolation.violation.message }
+                p { unsafe { +ruleViolation.violation.message } }
```

Comment:
After a discussion with @sschuberth I think using `unsafe` here is not a good idea, as it could easily break the HTML if special characters like `<` are used in the message. A better option would be to add a CSS class for `p` with [`white-space: pre-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space#values).

Reply:
Where is `p` defined?

Reply:
Just `Ctrl`+Click on it, it's built into *kotlinx-html*.

Reply:
I tried but it does not work on my machine, would you mind pasting here the link to the file in the public `kotlinx-html` repo? Thank you!

Reply:
Here you go: https://github.com/Kotlin/kotlinx.html/blob/c2f8ee0df425e5188681ac821034c8781ee2a146/src/commonMain/kotlin/generated/gen-tag-groups.kt#L125

Reply:
I implemented what (I think) you requested, but I do not see that CSS class being used anywhere in the HTML sources

Reply:
any ideas how to fix this?

Reply:
I implemented all suggested changes but the solution does not work, can someone please suggest a way forward? Thank you!

Reply:
> I do not see that CSS class being used anywhere in the HTML sources

I've just tested this and can confirm the CSS to *be* embedded the generated HTML. So, do you want to move this forward by also changing this line to use `p("ort-message")`, @bennati?

Reply:
Using `p("ort-message")` visualizes big indentations every time a multi-line string has a new line or any time two strings are joined with +. I tried with `trimMargins()` and `replaceIndentByMargin()` but it did not help.
I think I won't spend any more time on this PR, so we can just close this.